### PR TITLE
More prerequisites, and Getting Help

### PR DESCRIPTION
**Pull request by nedbat**

### CODE REVIEW

**Comment by nedbat** [comment]
@lamagnifica what do you think??

**Comment by nedbat** [comment]
@lamagnifica btw, Apple no longer says "Macintosh," it's all "Mac" with them now.

File: en_us/install_operations/source/installation/devstack/start_devstack.rst [hunk 1]
```diff
@@ -146,7 +146,7 @@ To run course discussions on devstack, follow these steps.
    .. note::
      If you get a message for entering a password to install the bundled
      RubyGems to the system, you can safely exit by entering ``control+c`` on a
-     Macintosh or ``Ctrl+C`` on Windows. The RubyGems will still be installed
+     Mac or ``Ctrl+C`` on Windows. The RubyGems will still be installed
```
Comment (lamagnifica) [inline en_us/install_operations/source/installation/devstack/start_devstack.rst:149]: Mac&#174; computer (that's & # 1 7 4 ; only without spaces)
per http://www.apple.com/legal/intellectual-property/trademark/appletmlist.html

Peter figured out that Sublime3 supports these, so if you have the bandwidth please add them...
(also ... on a Windows &#174; computer.)
https://www.microsoft.com/en-us/legal/intellectualproperty/trademarks/en-us.aspx#096c2b52-4010-4e38-9c79-59a828f1500e

File: en_us/install_operations/source/installation/getting_help.rst [hunk 2]
```diff
@@ -0,0 +1,50 @@
+.. _Getting Help:
+
+############
+Getting Help
+############
+
+During the installation process, or once your site is running, you may need
+technical help.
```
Comment (lamagnifica) [inline en_us/install_operations/source/installation/getting_help.rst]: perhaps, 
This section describes resources for getting help if you need technical assistance during the installation process or after your site is running.

("once" doesn't translate well, and we avoid "may" as it infers permission)

File: en_us/install_operations/source/installation/getting_help.rst [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+############
+Getting Help
+############
+
+During the installation process, or once your site is running, you may need
+technical help. 
+
+
+******************
+Where to find help
```
Comment (lamagnifica) [inline en_us/install_operations/source/installation/getting_help.rst]: title case: Where to Find Help
(throughout)
super nit

File: en_us/install_operations/source/installation/getting_help.rst [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+******************
+
+There are a number of places to get help, including mailing lists and real-time
+chat.  Please choose an appropriate venue for your question.  This helps ensure
+that you get good prompt advice, and keeps discussion focused.  The `Community
+Discussions`_ page has full details.
```
Comment (lamagnifica) [inline en_us/install_operations/source/installation/getting_help.rst]: standard format is 
For more information, see (ref). 
or, 
For more information about (subject), see (ref).

File: en_us/install_operations/source/installation/getting_help.rst [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+
+*******************************************
+Information to provide when asking for help
+*******************************************
+
+Keep in mind when asking for help that you have much more knowledge of your
+situation than others do.  You need to provide that context so that people can
+understand your problem and provide good help.  Here are kinds of information
```
Comment (lamagnifica) [inline en_us/install_operations/source/installation/getting_help.rst]: To make it easier for others to help you, be prepared to supply detailed information of the following types. (?)
then you can begin each bullet with the question: What are you trying to do? What have you already done? What went wrong?
or you could distill further to start with a summary word (for those who don't read past word 3 of a list item): **Goal**: What are you trying to do? **Status**: What have you already done? **Issue**: What went wrong?

File: en_us/install_operations/source/installation/getting_help.rst [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+* Details of what you have done.  What instructions were you following? What
+  commands did you run? Did you deviate from the instructions at any point,
+  even a little bit?  Full output of those commands, even if it seems
+  overwhelming, can be very useful.
+
+* Details of what has gone wrong.  Are there errors in log files? Did you get
+  specific failures in the terminal?  Provide full details about the undesired
+  results you saw.
+
+While working through your problem, helpers may want additional information.
```
Comment (lamagnifica) [inline en_us/install_operations/source/installation/getting_help.rst]: may want --> often request (?)

File: en_us/install_operations/source/installation/getting_help.rst [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+Stay involved in the discussion, and try to give them what they need. They know
+best the information they will need to solve the problem.
```
Comment (lamagnifica) [inline en_us/install_operations/source/installation/getting_help.rst]: the info... --> what info...

Comment (lamagnifica) [inline en_us/install_operations/source/installation/getting_help.rst]: strike "will"

File: en_us/install_operations/source/installation/installation_options.rst [hunk 8]
```diff
@@ -184,7 +184,7 @@ for edX virtual machines.
 Set Up Ability to Preview Units (Mac/Linux Only)
 ================================================
 
-If you are installing an edX virtual machine on a Linux or Macintosh computer,
+If you are installing an edX virtual machine on a Linux or Mac computer,
```
Comment (lamagnifica) [inline en_us/install_operations/source/installation/installation_options.rst:187]: Linux&#174; 
http://www.linuxfoundation.org/programs/legal/trademark/attribution

**Comment by lamagnifica** [comment]
Have no fear, SuperNit is here!
I love the tone of your getting help section, I think it might make a difference! 
:+1: as is, whatever of my edits you have time or inclination to do would be a nice to have.

File: en_us/install_operations/source/installation/devstack/start_devstack.rst (hunk 1 above)
Comment (nedbat) [inline en_us/install_operations/source/installation/devstack/start_devstack.rst:149]: I'm going to leave this for a future edit.

File: en_us/install_operations/source/installation/getting_help.rst (hunk 2 above)
Comment (nedbat) [inline en_us/install_operations/source/installation/getting_help.rst]: Done.

File: en_us/install_operations/source/installation/getting_help.rst (hunk 3 above)
Comment (nedbat) [inline en_us/install_operations/source/installation/getting_help.rst]: Done.

File: en_us/install_operations/source/installation/getting_help.rst (hunk 4 above)
Comment (nedbat) [inline en_us/install_operations/source/installation/getting_help.rst]: Updated.

File: en_us/install_operations/source/installation/getting_help.rst (hunk 5 above)
Comment (nedbat) [inline en_us/install_operations/source/installation/getting_help.rst]: I began each bullet with a bolded question.

File: en_us/install_operations/source/installation/getting_help.rst (hunk 6 above)
Comment (nedbat) [inline en_us/install_operations/source/installation/getting_help.rst]: "often need"

File: en_us/install_operations/source/installation/getting_help.rst (hunk 7 above)
Comment (nedbat) [inline en_us/install_operations/source/installation/getting_help.rst]: Done.

File: en_us/install_operations/source/installation/installation_prerequisites.rst [hunk 9]
```diff
@@ -14,13 +14,21 @@ Knowledge Prerequisites
...
   `Introduction to the Mac OS X Command Line`_. If you are using a Windows
   computer, see `Windows Command Line Reference`_.
 
 * Vagrant commands. See the `Vagrant Getting Started`_ guide for
   more information.
 
+* Diagnosing and fixing failures may involve many different technologies and
+  skills. It will help to know these things.
+
+  - The basics of how Python applications are built, installed and deployed.
```
Comment (jbarciauskas) [inline en_us/install_operations/source/installation/installation_prerequisites.rst:27]: "web applications"?

File: en_us/install_operations/source/installation/installation_prerequisites.rst [hunk 10]
```diff
@@ -14,13 +14,21 @@ Knowledge Prerequisites
...
+
+  - How to manage a Linux system, including supervisor.
```
Comment (jbarciauskas) [inline en_us/install_operations/source/installation/installation_prerequisites.rst:29]: and Ansible or other equivalent configuration management systems?

**Pull request merged**